package.json

```diff
@@ -56,5 +64,6 @@
       "type": "MIT",
       "url": "http://opensource.org/licenses/MIT"
     }
-  ]
+  ],
+  "false": {}
```

Comment:
So it was presumably added by npm automatically? I can't find docs for that. I wonder what it does.